.github/workflows/when-pr-is-closed.yml

```
@@ -0,0 +1,67 @@
+name: When a PR is closed
```

```suggestion
name: PR targeting releases branch is closed 
```

Maybe also rename the file `pr-targeting-releases-branch-is-closed.yml`?